src/Symfony/Bridge/Monolog/Handler/SwiftMailerHandler.php

```diff
@@ -21,6 +21,8 @@
  * @author Philipp Kräutli <pkraeutli@astina.ch>
  *
  * @final
+ *
+ * @deprecated in 5.4, will be removed in 6.0.
```

Comment:
fabbot should probably have reported this
```suggestion
 * @deprecated since Symfony 5.4
```